stubs/gcloud/aio/auth.pyi

```diff
@@ -0,0 +1,12 @@
+import aiohttp
```

Comment:
The lib actually includes typing, but it can not be found currently. I opened https://github.com/talkiq/gcloud-aio/issues/67 for it.